pkg/common/metadata/metadata.go

```diff
@@ -1,11 +0,0 @@
-/*
-Copyright SecureKey Technologies Inc. All Rights Reserved.
-SPDX-License-Identifier: Apache-2.0
-*/
-
-package metadata
-
-const (
-	// AriesCommunityDID aries community DID
-	AriesCommunityDID = "did:sov:BzCbsNYhMrjHiqZDTUASHg"
```

Comment:
Any specific reason to remove this common constant ? We moved to a common constant during [original commit](https://github.com/hyperledger/aries-framework-go/pull/19#discussion_r305093108) so that we can change the value easily across protocols.

Reply:
I found it wasn't used consistently - see rest of PR :-). Originally the constant was there anticipating the move to a https domain like didcomm.org.